api/v2/sdks/client-side-identity-v2.md

```diff
@@ -0,0 +1,298 @@
+[EUID Overview](../../../README.md) > [Getting Started: Summary](../getting-started/gs-summary.md) > [v2](../summary-doc-v2.md) > [SDKs](summary-sdks.md) > [SDK for JavaScript Reference Guide](./client-side-identity.md) > SDK for JavaScript Reference Guide (1.x)
+
+# SDK for JavaScript Reference Guide (1.x)
```

Comment:
while that would match the UID2 version, the documentation itself for version 1, rather than 2. i.e. this page wouldn't make sense for someone trying to use version 2.x. 

What would you suggest I do here? 

I'm inclined to leave as-is, though that would leave some disparity between EUID and UID2 docs.

Reply:
Hmm let's see what LP says then.

Reply:
Ended up going with your suggestion - thanks!